src/docs/product/relay/options.mdx

```diff
@@ -6,34 +6,42 @@ redirect_from:
   - /meta/relay/options/
 ---
 
-Configuration for Relay is recorded in the file `.relay/config.yml`. To change this location, pass the `--config` option to any Relay command:
+Configuration for Relay is recorded either in the file `.relay/config.yml`, or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:
 
 ```shell
 ❯ ./relay run --config /path/to/folder
 ```
 
 All configuration keys are `snake_case`.
 
+## Credentials
+
+Credentials can be initiated as described in the [getting started page](https://sentry-docs-git-relayconfig.sentry.dev/product/relay/getting-started/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-docs%2Fpull%2F7062#creating-credentials), but can be overriden by setting the following environment variables:
```

Comment:
I don't know what's the best practice for links here, but you don't _need_ to write the full URL, see:
https://github.com/getsentry/sentry-docs/blob/55c4d20aaded7449ac21ec792f615a541952293e/src/platforms/android/using-ndk.mdx?plain=1#L19
I think we can get rid of the referrer parameter too?
@lizokm may be able to provide some guidance here.